pyodide-build/pyodide_build/cli/build.py

```diff
@@ -151,6 +158,47 @@ def main(
     elif Path(source_location).is_dir():
         # a folder, build it
         wheel = source(source_location, exports, ctx)
+    elif source_location.lower().endswith(".txt"):
```

Comment:
Since a package name can actually endswith ".txt" (though it is not very recommended...). I think we could use a `-r` flag for requirements files same as pip.